lib/view_models/records/record_tag_filter.dart

```diff
@@ -11,18 +12,32 @@ class RecordTagFilterViewModel extends ChangeNotifier {
   /// [RecordService] used to load data for the tag filter.
   final RecordService _service = RecordService.getInstance();
 
+  final SecureStorageService _storage = SecureStorageService.getInstance();
+
   /// Initializes the view model.
   RecordTagFilterViewModel(this.tagFilter);
 
   /// Clears the filter value of the [identifier].
   void clear(String identifier) {
     tagFilter.clear(identifier);
+     if (identifier == ID3TagFilters.folderView) {
+     _storage.set(
```

Comment:
await?

Reply:
Should here really the folder view be resetted

Reply:
Yes.